cmd/remoteenforcer/remoteenforcer_linux.go

```diff
@@ -94,34 +95,9 @@ func (s *Server) InitEnforcer(req rpcwrapper.Request, resp *rpcwrapper.Response)
 		return errors.New(resp.Status)
 	}
 
-	pid := strconv.Itoa(os.Getpid())
```

Comment:
Resurrect this when not doing private mount as this is the more common code path and this check helps.